strong_migrations.gemspec

```diff
@@ -16,4 +16,9 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 3.1"
 
   spec.add_dependency "activerecord", ">= 6.1"
+
+  spec.metadata = {
+    "changelog_uri"     => "https://github.com/ankane/strong_migrations/blob/v#{StrongMigrations::VERSION}/CHANGELOG.md",
+    "source_code_uri"   => "https://github.com/ankane/strong_migrations",
```

Comment:
I was tempted also to add a `documentation_uri` which links to `"https://github.com/ankane/strong_migrations/blob/v#{StrongMigrations::VERSION}/README.md"`. The default documentation link goes to https://www.rubydoc.info/gems/strong_migrations/2.0.1, which also works fine. But as a user of the gem, I'm less interested in the internals. I just want the content in the README.